src/assets/styles/01-settings/global.scss

```diff
@@ -4,6 +4,8 @@ $main-color: var(--color-primary);
 $infinte-color: var(--infinte-color);
 $main-text-color: var(--main-text-color);
 
+//direction
+$direction: rtl;
```

Comment:
how you can change the direction?

we know the direction using this: `[dir="rtl"]`

Reply:
resolved and  noted